src/main/java/mate/academy/spring/validation/PasswordValidator.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.spring.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import mate.academy.spring.dto.request.UserRequestDto;
+
+public class PasswordValidator implements ConstraintValidator<Password, UserRequestDto> {
+    @Override
+    public boolean isValid(UserRequestDto userRequestDto, ConstraintValidatorContext context) {
+        return userRequestDto.getPassword() != null;
```

Comment:
There is no point in having a custom validator which is only checking for null.
Since this part of the task is optional we should either remove it or add a `repeatPassword` validation